### PR TITLE
Meta & Delta Medbay rework

### PR DESCRIPTION
**Pull request by BriggsIDP**
Copy of this: https://github.com/BeeStation/BeeStation-Hornet/pull/8906

## Medbay redesign

- [x] Remap meta medbay
- [x] Copy TG code for stasis bed directions

This pull request consists of a complete remake of the medbay area, focusing on the treatment area but also including the security post, virology department, genetics department, and other supplementary departments.

## Why It's Good For The Game

Better layout and design: The updated layout provide a more intuitive and user-friendly experience for players, making it easier for them to navigate, expand, and utilize the medbay facilities.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

image

image

image

</details>

## Changelog
:cl:
tweak: tweaked DeltaStation2.dmm
tweak: tweaked MetaStation2.dmm
tweak: added a jani room to medbay that CMO can access
add: Custodian zoning
add: Extra Stasis direction
/:cl:

### CODE REVIEW

**Comment by EvilDragonfiend** [comment]
please relocate virus fridge. chem master doesn't have to be next to their workdesk but the fridge should.

**Comment by BriggsIDP** [comment]
> please relocate virus fridge. chem master doesn't have to be next to their workdesk but the fridge should.
both maps where already updated with this feedback! 
image
image

**Comment by EvilDragonfiend** [comment]
ah cool. the image was outdated.

**Comment by PowerfulBacon** [comment]
I like the look of it, but we need to be careful that we don't create more medbays like Box medbay where it is far too optimised and there are 8 people in the lobby with the rest of medical being a wasteland. From the screenshots this seems like it would be fine, but we can put it in and test.

**Comment by itsmeow** [comment]
Very cool. I would like more maints though, it seems this PR has reduced the amount of rooms a bit

File: code/game/machinery/stasis.dm [hunk 1]
```diff
@@ -19,17 +18,32 @@
...
 	return INITIALIZE_HINT_LATELOAD
 
 /obj/machinery/stasis/LateInitialize()
 	. = ..()
 	initial_link()
+	for(var/direction in GLOB.alldirs)
+		op_computer = locate(/obj/machinery/computer/operating) in get_step(src, direction)
+		if(op_computer)
+			op_computer.sbed = src
+			src.op_computer = op_computer
```
Comment (itsmeow) [inline code/game/machinery/stasis.dm]: I think this is reverting a change made in another PR?

Comment (BriggsIDP) [reply code/game/machinery/stasis.dm]: Thanks for the spot, should be resolved

**Comment by EvilDragonfiend** [comment]
git checkout origin/master code/game/machinery/stasis.dm
git checkout origin/master icons/obj/machines/stasis.dmi

use this command to revert unwanted change

**Comment by BriggsIDP** [comment]
As talked about, the stasis changes are intentional and documented in the body.

**Review by PestoVerde322 — APPROVED** [review]
All good to go, approved.

**Pull request merged**